aws/v11.4.0/release-notes.md

```diff
@@ -0,0 +1,56 @@
+## :zap: Giant Swarm Release v11.4.0 for AWS :zap:
+
+This release introduces master node high availability, which means that clusters will have three master nodes by default in three different availability zones (or ).
+
+- Multiple master nodes are only supported in AWS regions having at least three availability zones. This excludes China/Beijing (`cn-north-1`).
+
+**Note for SEs:** This release contains an external-dns fix introduced in [11.3.2](https://github.com/giantswarm/releases/blob/master/aws/v11.3.2/release-notes.md). It requires manual intervention for cluster upgrades in China to work. When upgrading a cluster, existing ingress A+TXT record sets do not get replaced with CNAME+TXT record sets even when external-dns is configured with CNAMEs as preferred. After upgrading, delete the ingress A+TXT record sets. external-dns will then automatically create CNAME+TXT record sets.
+
+**Note for future 11.3.x releases:** Until upstream external-dns issue is fixed, please include this note and the one above in all future 11.3.x releases.
```

Comment:
@cokiengchiara Are these notices actually required in 11.4.x?

Reply:
I think yes, for China cluster upgrades (but not creation) from 11.3.x to 11.4.0. Can @sslavic confirm?

Reply:
@sslavic Would be nice if you could confirm this, as we are approaching the release

@cokiengchiara I guess the `Note for future 11.3.x releases` part needs adaptation to 11.4.x, right?

Reply:
> @cokiengchiara I guess the `Note for future 11.3.x releases` part needs adaptation to 11.4.x, right?

If I understand correctly, yes. But would love @sslavic to confirm. Just pinged him in Slack about it too.

Reply:
@marians @cokiengchiara 

Note is applicable if:
- AWS platform release 11.4.0 makes use of external-dns-app v1.2.1, and
- Customer can upgrade an existing cluster from older node pools 10+ platform release with older external-dns-app, to 11.4.0, and if
- Cluster being upgraded is in AWS China (currently limited to few Adidas clusters).

external-dns-app v1.2.1 introduced a fix to a problem Adidas was experiencing, external-dns configuration was changed to prefer CNAME. It works automatically (creates route53 records) for the new clusters, but for cluster upgrades CNAME preference is not respected (existing route53 records do not get automatically updated, have to be manually dropped first, and then external-dns creates new valid ones) - therefore the note with instructions how to get if fixed, and also meta-note to transfer the note to a future release notes too.

Issue has been reported upstream, https://github.com/kubernetes-sigs/external-dns/issues/1617
It's not getting any traction from the community. Likely it's hurting us the most. We should consider investing time and effort to contribute the fix.

Maybe another alternative is to ensure together with @jgsqware that no more clusters get created with AWS [10 - 11.3.1], so without CNAME preference. Then we need also to ensure all the existing clusters in AWS China are upgraded to a newer release and have the manual workaround applied. If both of that happens, proactively or eventually, I guess we can drop the note about manual upgrade step even before/without upstream bug fix.

Reply:
thanks, then the answer for if this note should persist is yes. @sslavic regarding us working on external dns, provided my thoughts here https://github.com/giantswarm/giantswarm/issues/11606

Reply:
Thanks!